src/libecalc/presentation/yaml/yaml_types/emitters/yaml_venting_emitter.py

```diff
@@ -222,7 +223,9 @@ def get_emissions(
                 .tolist()
             )
             emission_rate = [oil_rate * factor for oil_rate, factor in zip(oil_rates, factors)]
-            emission_rate = Unit.to(self.volume.rate.unit, Unit.TONS_PER_DAY)(emission_rate)
+
+            # Emission factor is kg/Sm3 and oil rate/volume is Sm3/d. Hence, the emission rate is in kg/d:
+            emission_rate = Unit.to(Unit.KILO_PER_DAY, Unit.TONS_PER_DAY)(emission_rate)
```

Comment:
```suggestion
            emission_rate = Unit.KILO_PER_DAY.to(Unit.TONS_PER_DAY)(emission_rate)
```
Covert to TONS_PER_DAY, right?

Reply:
Yes:-)